### PR TITLE
Added dectate 0.13

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

File: recipes/dectate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: Please take a look at the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

and attempt to conform this recipe as closely as possible to that guide.

**Comment by stuertz** [comment]
@conda-forge/core It is ready for merge.

File: recipes/dectate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: I believe that the `preserve_egg_dir` should be deleted, although I'm not 100% sure. I have not seen it for any other recipes as far as I can remember.

File: recipes/dectate/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: I do not believe that `setuptools` is required to run this package. Please double check.

File: recipes/dectate/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  # Python imports
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: Delete comment.

File: recipes/dectate/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - dectate
+    - dectate.tests
+    - dectate.tests.fixtures
+
+about:
+  home: http://dectate.readthedocs.io
+  license: BSD
+  summary: 'A configuration engine for Python frameworks'
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: Please add:

```
license_file: LICENSE.txt
...
dev_url: https://github.com/morepath/dectate
doc_url: http://morepath.readthedocs.org
```

File: recipes/dectate/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+  run:
+    - python
+    - setuptools
+
+test:
+  # Python imports
+  imports:
+    - dectate
+    - dectate.tests
```
Comment (synapticarbors) [inline recipes/dectate/meta.yaml]: Typically, you don't have to import the test modules.

File: recipes/dectate/meta.yaml (hunk 5 above)
Comment (stuertz) [inline recipes/dectate/meta.yaml]: Thanks for the review, I've updated the PR.

Comment (stuertz) [inline recipes/dectate/meta.yaml]: done.

File: recipes/dectate/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - dectate
+
+about:
+  home: http://dectate.readthedocs.io
+  license: BSD
```
Comment (ocefpaf) [inline recipes/dectate/meta.yaml]: Version?

Comment (stuertz) [inline recipes/dectate/meta.yaml]: its bsd-3. fixed.

**Pull request merged**